Makefile

```diff
@@ -4,12 +4,14 @@ MAKE := make
 RM := rm
 INSTALL := install
 GIT_VERSION := $(shell git describe --abbrev=6 --dirty --always --tags)
+INC_DIR = include
+# DEPS = $(INC_DIR)/common.h
 
 all: beagle-tester
 
 beagle-tester: beagle-tester.c
```

Comment:
Just put $DEPS on the right hand side.

Reply:
How do other makefiles handle dependencies without explicitly stating what files to use?